frontend/src/app/modules/queue-dashboard/queue-dashboard.component.html

```diff
@@ -0,0 +1,34 @@
+<h1>
+  {{'frontend.de.iteratec.osm.queueDashboard.title' | translate}}
+</h1>
+<div class="card" *ngFor="let wptserver of wptServer$ | async" #thisCard isOpened="false" isLoaded="false">
+
+  <div>
+    <span>
+      <h2>{{wptserver.label}}</h2>
+      <span class="fas chevron-large"
+          (click)="thisCard.isOpened = !thisCard.isOpened"
+          [style.display]="thisCard.isLoaded ? 'block' : 'none'"
```

Comment:
Same as in the other component: Why setting the `display` style instead of using this in an `*ngIf` directive?